pygame 2.1.2 (SDL 2.0.16, Python 3.10.8)
Hello from the pygame community. https://www.pygame.org/contribute.html
Press the red sync button on the board now
Found WiiBoard at address 00:1F:C5:9E:1E:24
Connected to WiiBoard at address 00:1F:C5:9E:1E:24
Requesting calibration
WiiBoard connected
Traceback (most recent call last):
  File "/home/capdz/waverider/run_game.py", line 12, in <module>
    main.main(mainpath)
  File "/home/capdz/waverider/src/main.py", line 39, in main
    dir.change_scene('titlescreen', [])
  File "/home/capdz/waverider/src/gamedirector.py", line 66, in change_scene
    self.scene.on_switchto(switchtoargs)
  File "/home/capdz/waverider/src/game.py", line 373, in on_switchto
    board.start_service()
AttributeError: 'Wiiboard' object has no attribute 'start_service'
